### PR TITLE
Removed explicit registration of hidden assemblies for callsite handling

### PR DESCRIPTION
**Pull request by snakefoot**
NLog.Extensions.Logging ver. 1.0.1 now handles everything in its NLogLoggerProvider.

And also corrected obsolete notifications for ConfigureNLog to match the Wiki

### CODE REVIEW

**Comment by snakefoot** [comment]
@304NotModified Have updated to NLog.Extensions.Logging ver. 1.0.1, and adjusted the code accordingly.

File: NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
     <TargetFrameworks>netstandard1.5;net451;net461;netstandard2.0</TargetFrameworks>
 
-    <VersionPrefix>4.5.1</VersionPrefix>
+    <VersionPrefix>4.5.2</VersionPrefix>
```
Comment (304NotModified) [inline NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj]: Please update the release notes (and maybe also the changelog) when updating the version (cannot be done afterwards anymore :(.     )
Or leave the version unchanged, thanks!

Comment (snakefoot) [reply NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj]: Reverted change

**Comment by 304NotModified** [comment]
Thanks!

**Pull request merged**